ironfish/src/primitives/target.test.ts

```diff
@@ -4,6 +4,12 @@
 
 import { bigIntToBytes, bytesToBigInt, Target, TargetSerde } from './target'
 
+/**
```

Comment:
This comment is on the `Describe` for all target tests, but it talks about target calculation. Who is this intended for exactly? Someone trying to learn the target calculation? Or debugging a test and trying to fix the tests?